cmd/kube-apiserver/app/options/options.go

```diff
@@ -130,11 +131,11 @@ func gvToMap(gvList string, dest map[string]string) {
 		// "group=group/version". The latter is used when types
 		// move between groups.
 		if !strings.Contains(gv, "=") {
-			dest[apiutil.GetGroup(gv)] = gv
+			dest[apiutil.GetGroup(gv)] = unversioned.GroupVersion{Group: apiutil.GetGroup(gv), Version: apiutil.GetVersion(gv)}
```

Comment:
> Sorry I did not understand why cant we use ParseGroupVersion from

It's just a change that will start rejecting group versions that used to be accepted.  Things like foo/bar/baz are parsed (badly) by the `apiutil` methods in addition to other improper input.  I have no issue with change it, other than that it may break previously "working" users.